### DISCUSSION TITLE
Grouping restapi and mongodb query

### DISCUSSION THREAD

**Discussion by eisenhaus335** [Budibase/budibase#3813] [Ideas]
I want to group my query so i could find it easily but seems there are no ways for this. It might nice feature to be added to help development

**Comment by mjashanks** [comment]
Ah - you mean like in a directory in here?

![image](https://user-images.githubusercontent.com/3524181/146558343-140869b6-4d7a-4230-afba-79f067ac9460.png)

**Reply by eisenhaus335** [reply]
This is neat. Look fine to me. Thanks!
